Core/PARStore.m

```diff
@@ -1458,8 +1468,11 @@ - (void)_sync
     
     // just go through each row (back in time) until all entries are loaded
     NSMutableDictionary *updatedValues = [NSMutableDictionary dictionary];
-    for (NSManagedObject *log in allLogs)
+    for (NSManagedObjectID *logID in allLogObjectIDs)
     {
+        // Get object
+        NSManagedObject *log = [moc objectWithID:logID];
```

Comment:
Isn't this resulting in n database fetches instead of a single one before? The problem here is that in practice we almost always have to go back to the very beginning on each initial load (as uuid is the very first row)